src/edu/csus/ecs/pc2/clics/API202306/CLICSScore.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.standings.TeamStanding;
+
+/**
+ * CLICS team score
+ * Contains information the score for a team
+ * 
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSScore {
+
+    @JsonProperty
+    private int num_solved;
+    
+    @JsonProperty
+    private int total_time;
+
+// Not used for pass/fail contest and we don't want it appearing in the json
+//    @JsonProperty
+//    private int score;
+
+    @JsonProperty
+    private String time;
+
+    /**
+     * Fill in the properties for a team's score
+     * 
+     * @param teamStanding The team's scoring information
+     */
+    public CLICSScore(TeamStanding teamStanding) {
+        num_solved = Utilities.nullSafeToInt(teamStanding.getSolved(), 0);
+        total_time = Utilities.nullSafeToInt(teamStanding.getPoints(), 0);
```

Comment:
If `teamStanding.getPoints()` is null or empty, `total_time` is set to zero (the specified "default").  A time of "zero" is "Very Good" (means the team gets zero penalty points).  I wonder if the default shouldn't be a LARGE number, or if instead this assignment should be conditioned on something, like maybe `num_solved`?

Reply:
I am not inclined to change this since this code was "borrowed" from other modules, and, since the TeamStanding object is deserialized XML from XML that was created by the DSA (or NSA - I'm not sure which), it better have correct values in it.